go.mod

```diff
@@ -4,9 +4,9 @@ go 1.19
 
 require (
 	github.com/docker/go-units v0.4.0
-	github.com/go-openapi/runtime v0.24.1
+	github.com/go-openapi/runtime v0.24.2
 	github.com/go-openapi/strfmt v0.21.3
-	github.com/signadot/go-sdk v0.3.3
+	github.com/signadot/go-sdk v0.3.4
```

Comment:
I had pushed the v0.3.4 tag without merging in the code generated with the latest swagger.yaml. I did that afterwards and updated the tag to point to the latest commit with generated code. But pkg.go.dev still has the previous version at the moment.

https://pkg.go.dev/github.com/signadot/go-sdk@v0.3.4/models#SandboxSpec

<img width="861" alt="image" src="https://user-images.githubusercontent.com/3791487/202039410-725c8a3d-439d-4b7d-88f8-b6968bd4b725.png">

With this, the sandboxes created with the newly generated CLI isn't getting created with labels (though the spec has labels defined).